src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -709,6 +710,16 @@ public class SimpleEvents {
 					.requiredPlugins("Paper 1.16+");
 		}
 
+		Skript.registerEvent("Inventory Drag", SimpleEvent.class, InventoryDragEvent.class, "inventory drag[ging]")
+			.description("Called when a player drags an item in their cursor across the inventory.")
+			.examples(
+				"on inventory drag:",
+				"\tif player's current inventory is {_gui}",
+				"\t\tsend \"You can't drag your items here!\" to player",
+				"\t\tcancel event"
+			)
+			.since("INSERT VERSION");
```

Comment:
```suggestion
				.description("Called when a player drags an item in their cursor across the inventory.")
				.examples(
						"on inventory drag:",
						"\tif player's current inventory is {_gui}",
						"\t\tsend \"You can't drag your items here!\" to player",
						"\t\tcancel event"
				)
				.since("INSERT VERSION");
```